config/sync/node.type.vamc_system_va_police.yml

```
@@ -0,0 +1,169 @@
+      - va-madison-health-care
+      - va-maine-health-care
+      - va-manchester-health-care
+      - va-marion-health-care
+      - va-martinsburg-health-care
+      - va-maryland-health-care
+      - va-memphis-health-care
+      - va-miami-health-care
+      - va-milwaukee-health-care
+      - va-minneapolis-health-care
+      - va-montana-health-care
+      - va-mountain-home-health-care
+      - va-nebraska-western-iowa-health-
+      - va-new-jersey-health-care
+      - va-new-mexico-health-care
+      - va-new-york-harbor-health-care
+      - va-north-florida-health-care
+      - va-north-texas-health-care
+      - va-northeast-ohio-health-care
+      - va-northern-arizona-health-care
+      - va-northern-california-health-ca
+      - va-northern-indiana-health-care
+      - va-northport-health-care
+      - va-oklahoma-health-care
+      - va-orlando-health-care
+      - va-pacific-islands-health-care
+      - va-palo-alto-health-care
+      - va-philadelphia-health-care
+      - va-phoenix-health-care
+      - pittsburgh-health-care
+      - va-poplar-bluff-health-care
+      - va-portland-health-care
+      - va-providence-health-care
+      - va-puget-sound-health-care
+      - va-richmond-health-care
+      - va-roseburg-health-care
+      - va-saginaw-health-care
+      - va-salem-health-care
+      - va-salisbury-health-care
+      - va-salt-lake-city-health-care
+      - va-san-diego-health-care
+      - va-san-francisco-health-care
+      - va-sheridan-health-care
+      - va-shreveport-health-care
+      - va-sierra-nevada-health-care
+      - va-sioux-falls-health-care
+      - va-south-texas-health-care
+      - va-southeast-louisiana-health-ca
+      - va-southern-arizona-health-care
+      - va-southern-nevada-health-care
+      - va-southern-oregon-health-care
+      - va-spokane-health-care
+      - va-st-cloud-health-care
+      - va-st-louis-health-care
+      - va-syracuse-health-care
+      - va-tampa-health-care
+      - va-tennessee-valley-health-care
+      - va-texas-valley-health-care
+      - va-tomah-health-care
+      - va-topeka-health-care
+      - va-tuscaloosa-health-care
+      - va-walla-walla-health-care
+      - va-washington-dc-health-care
+      - va-west-palm-beach-health-care
+      - va-west-texas-health-care
+      - va-western-colorado-health-care
+      - va-western-new-york-health-care
+      - va-white-river-junction-health-c
+      - va-wichita-health-care
+      - va-wilkes-barre-health-care
+      - va-wilmington-health-care
+    parent: 'va-alaska-health-care:'
+  node_revision_delete:
+    minimum_revisions_to_keep: 50
+    minimum_age_to_delete: 0
+    when_to_delete: 0
+  node_title_help_text:
+    title_help: ''
+  menu_force:
+    menu_force: true
+    menu_force_parent: false
+name: 'VAMC System VA Police page'
+type: vamc_system_va_police
+description: 'A page about how Veterans see VA Police information for a VAMC system.'
+help: ''
+new_revision: true
+preview_mode: 1
```

```suggestion
preview_mode: 0
```

Good catch @dsasser Thank you for spotting it.